test/test_nn.py

```diff
@@ -18295,6 +18295,45 @@ def test_cross_entropy_label_smoothing_with_probs(self, device):
                 self.assertEqual(output_with_smoothing, output_with_manual_smoothing)
 
 
+    def test_cross_entropy_label_smoothing_weight_ignore_indices(self, device):
+        reductions = ['none', 'mean', 'sum']
```

Comment:
I think it might be better to have a test validating equal outputs for:
* an input with 2 samples, 1 ignored
* an input with 1 sample (same as non-ignored one above)

This test would find the denominator problem